src/pages/experimental/edit-pattern/index.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Edit Pattern
+---
+
+<PageDescription>
+
+Editing is making changes to a resource that already exists in the system.
+Updating and editing should use the same behaviour as when the resource is created. This should only be varied in situations where the number of editable fields is significantly smaller or larger than during creation, or the context of current settings is required.
+
+Warn the user when the consequence of editing could damage resources or connections in their system.
+
+</PageDescription>
+
+#### Status:
+
+[Experimental](/experimental/overview/)
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.com/vikkipaterson), [Chloe Poulter](https://github.com/chloepoulterdesign)
+
+### Options:
```

Comment:
```suggestion
### Options:
```
We can't have headers above the anchorlinks here. As you mention that the one should consider the consequences before allowing a deletion, I think we can just delete this header/line below.

Reply:
So just remove this line?